### ISSUE TITLE
start/finish session button

### ISSUE THREAD

**Issue by erosson** [mapwatch/mapwatch#8]
> To do this, the tool would need some kind of "start/finish a session" button

as opposed to the built-in "today" session. Seems reasonable - the midnight cutoff is a bit awkward, and timing different approaches this way could work well. I don't like manual steps, but this is passable since it's not done while mapping.

there's a few ways to do this, not mutually exclusive. TODO: decide which one(s) are best
* magic chat commands. big advantage: they're included and persisted with the rest of our main data source; no need to alt-tab. disadvantage: non-obvious, might encourage people to micromanage their sessions.
* start/finish buttons on the website. we'd have to start using localstorage/some other form of persistence. easy to learn/teach.
* pick timestamps in the history to split up sessions after-the-fact. this would be nice to have for me testing, but probably less practical for realtime users.
* #5's search-by-date-range; a "session" is just a url with a date range search, nothing new to persist (though there might be start/finish buttons or other such ceremony to create that url)

https://www.reddit.com/r/pathofexile/comments/8lnctd/mapwatch_a_new_tool_to_automatically_time_your/dzh8rgi/

**Comment by erosson** [comment]
this is working, hidden behind `?enableSession=1`.

the session buttons create timestamped urls. starting a session hides all earlier runs; ending one hides all later runs, leaving you with the runs between those two times. no persistence or fancy state-juggling needed.

I deliberately avoided the word "session" - it sounded persistent, and also like you *have* to press it before starting. current ui is "hide earlier maps"/"snapshot current history".

**Comment by erosson** [comment]
Released in https://github.com/mapwatch/mapwatch/commit/7133419b6531e37539c12e3c97d00395db273839
